src/disclosures/js/utils/query-handler.js

```diff
@@ -11,28 +11,38 @@ function queryHandler( queryString ) {
   var valuePairs = {};
   var parameters = {};
   var numericKeys = [
-    'tf', 'rb', 'bk', 'tr', 'oe', 'sc', 'pg', 'sv', 'ws', 'fm', 'fl',
-    'pk', 'ss', 'su', 'il', 'ir', 'pl', 'pr'
+    'iped', 'pid', 'oid', 'tuit', 'hous', 'book', 'tran', 'othr',
+    'pelg', 'schg', 'stag', 'othg', 'mta', 'gib', 'wkst', 'parl',
+    'perl', 'subl', 'unsl', 'ppl', 'gpl', 'prvl', 'prvi', 'insl', 'insi', 'sav'
   ];
   var keyMaps = {
-    tf : 'tuitionFees',
-    rb : 'roomBoard',
-    bk : 'books',
-    tr : 'transportation',
-    oe : 'otherExpenses',
-    sc : 'scholarships',
-    pg : 'pell',
-    sv : 'savings',
-    ws : 'workstudy',
-    fm : 'family',
-    fl : 'state529plan',
-    pk : 'perkins',
-    ss : 'staffSubsidized',
-    su : 'staffUnsubsidized',
-    il : 'institutionalLoan',
-    ir : 'institutionalLoanRate',
-    pl : 'privateLoan',
-    pr : 'privateLoanRate'
+    iped: 'collegeID',
+    pid:  'programID',
+    oid:  'offerID',
+    tuit: 'tuitionFees',
+    hous: 'roomBoard',
+    book: 'books',
+    tran: 'transportation',
+    othr: 'otherExpenses',
+    pelg: 'pell',
+    schg: 'schoolGrants',
+    stag: 'state529plan',
+    othg: 'otherScholarships',
+    mta:  'militaryAssistance',
+    gib:  'giBill',
+    fam:  'familyContrib',
+    wkst: 'workstudy',
+    parl: 'parentLoan',
+    perl: 'perkins',
+    subl: 'staffSubsidized',
+    unsl: 'staffUnsubsidized',
+    ppl:  'parentplus',
+    gpl:  'gradplus',
+    prvl: 'privateLoan',
+    prvi: 'privateLoanRate',
+    insl: 'institutionalLoan',
+    insi: 'institutionalLoanRate',
+    sav:  'savings'
```

Comment:
This is the only var I don't see in the url spec. Otherwise, looks perfect. :+1: